ci/prow/config.yaml

```diff
@@ -3693,7 +3935,7 @@ periodics:
       - "--scenario=kubernetes_execute_bazel"
       - "--clean"
       - "--job=$(JOB_NAME)"
-      - "--repo=github.com/knative/sample-controller"
+      - "--repo=knative.dev/sample-controller"
```

Comment:
This is wrong.

Reply:
what should this be?

Reply:
AFAIU github.com/knative

Reply:
I think that will clone into `[GOPATH]/src/github.com/knative`, and we want `[GOPATH]/src/knative.dev`

Reply:
obsolete